src/labone/core/session.py

```diff
@@ -82,6 +85,8 @@ class NodeInfo(TypedDict):
     Type: NodeType
     Unit: str
     Options: NotRequired[dict[str, str]]
+    GetParser: NotRequired[t.Callable[[T], T]]
+    SetParser: NotRequired[t.Callable[[T], T]]
```

Comment:
issue: The parser stuff must not be mentioned anywhere in the core. Since it does not come from the data server.  I would clearly separate the node info and the parser stuff (also in the nodetree module)